### PR TITLE
Absolutize exercise urls in reference view pages API

### PR DESCRIPTION
**Pull request by karenc**
- Move update links in book code in UpdatePageContent into a function
- Update exercise links to absolute urls in reference view pages API

### CODE REVIEW

**Comment by karenc** [comment]
Ready for review.

**Comment by philschatz** [comment]
Can we wait to merge until there is a corresponding FE PR? Right now exercises work thanks to CORS

File: spec/subsystems/content/routines/update_page_content_spec.rb [hunk 1]
```diff
@@ -68,4 +77,41 @@
...
   end
+
+  it 'updates exercise links to absolute urls' do
+    doc = Nokogiri::HTML(page_with_exercises.content)
+
+    expected_links = doc.xpath('//a[starts-with(@href, "#ost")]').collect do |link|
+      link.attribute('href').value
+    end
+
+    # Remove the exercise linked to apbio-ch02-ex031
```
Comment (jpslav) [inline spec/subsystems/content/routines/update_page_content_spec.rb]: Can you clarify what's going on here with these destroys?  I don't follow what's being tested.

Comment (karenc) [inline spec/subsystems/content/routines/update_page_content_spec.rb]: Added more comments......

**Pull request merged**